### PR TITLE
Final Header update v1.0

### PR DESCRIPTION
**Pull request by swapnil2308**

### CODE REVIEW

**Comment by ajinkyaa** [comment]
where is your bootstrap js and jquery js? add that to the end of the file

**Comment by ajinkyaa** [comment]
start sending the PR from the start of the project, dont send it directly in the middle

File: 26.8.2015/css/main.css [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
-	
-}
\ No newline at end of file
+	padding: 30px 0;
+	background-color: #000;
+	font-size: 16px;
+}
+header .logo{
+	background-image: url(../img/icons.png);
+	background-position: 0 0;
```
Comment (ajinkyaa) [inline 26.8.2015/css/main.css]: indentation

**Comment by ajinkyaa** [comment]
make it 2 spaces indentation

File: 26.8.2015/css/main.css [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+}
+header .signUp:hover{
+	background-color: rgba(0, 255, 0, 0.2);
+}
+
+header .signUp:focus{
+	background-color: transparent;
+}
+
+/* Banner Will Start Here */
```
Comment (ajinkyaa) [inline 26.8.2015/css/main.css]: "ensure_newline_at_eof_on_save": true

add this to your sublime user settings

**Comment by yogesh-webonise** [comment]
## Points Need to consider:
1. Added extra css in css folder which can increase unnecessary space of project.
2. Use 2 space indentation.

@swapnil2308  @ajinkyaa

**Comment by ajinkyaa** [comment]
@yogesh-webonise , @swapnil2308 : this branch looks fine to me merging this branch, for the stable branch i am merging the branch

**Pull request merged**